receiver/otlpjsonfilereceiver/file_test.go

```diff
@@ -35,60 +36,6 @@ func TestDefaultConfig(t *testing.T) {
 	require.NoError(t, componenttest.CheckConfigStruct(cfg))
 }
 
-func TestFileTracesReceiver(t *testing.T) {
```

Comment:
Is there a reason you are deleting this test?

Reply:
this test is incorporated into `TestFileReceiver` as a parameterized test